src/cf-typography/src/licensed-fonts.less

```diff
@@ -0,0 +1,103 @@
+/*
+ * Source: http://fast.fonts.net/cssapi/44e8c964-4684-44c6-a6e3-3f3da8787b50.css
+ * This file has been edited to use absolute URLS so we can concatenate it with
+ * all of our other styles.
+ */
```

Comment:
I think this whole comment can be removed, but add some kind of general header banner for the file.